kimera_semantics/src/semantic_integrator_base.cpp

```diff
@@ -325,9 +344,9 @@ void SemanticIntegratorBase::normalizeProbabilities(
   if (normalization_factor != 0.0) {
     unnormalized_probs->normalize();
   } else {
-    CHECK_EQ(unnormalized_probs->size(), kTotalNumberOfLabels);
+    CHECK_EQ(unnormalized_probs->size(), semantic_config_.total_number_of_labels);
     static const SemanticProbability kUniformLogProbability =
-        std::log(1 / kTotalNumberOfLabels);
+        std::log(1 / semantic_config_.total_number_of_labels);
```

Comment:
Not sure why I didn't have a float division there... but I think it should be:
```suggestion
        std::log(1.0 / float(semantic_config_.total_number_of_labels_);
```